src/js-module.ts

```diff
@@ -32,14 +32,22 @@ export interface ConversionResult {
   readonly originalUrl: OriginalDocumentUrl;
   readonly convertedUrl: ConvertedDocumentUrl;
   readonly convertedFilePath: ConvertedDocumentFilePath;
+
   /**
    * Explicitly keep or remove the original file from disk. By default, the
    * original file will be destroyed. If the convertedFilePath matches
    * originalUrl, the original file will always be overwritten with the
    * converted output.
    */
   readonly deleteOriginal?: boolean;
-  readonly output: HtmlFile|JsModule;
+
+  /**
+   * An object describing the converted output of this module, or `undefined`.
+   *
+   * If `undefined`, the file is deleted regardless of the value of
```

Comment:
mention that no new file is created either, that was the 1 tricky big I got while reading this.

Reply:
ok